src/ProductView/Diagram/LineChart.js

```diff
@@ -51,6 +56,7 @@ class LineChart extends Component {
     componentDidUpdate() {
         const charWrapper = this.refs.canvas;
         if (charWrapper) {
+            console.log(this.buildChartData());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS2228)